Assets/Mirror/Runtime/NetworkConnection.cs

```diff
@@ -208,10 +208,11 @@ public virtual bool Send(int msgType, MessageBase msg, int channelId = Channels.
         /// <param name="msg">The message to send.</param>
         /// <param name="channelId">The transport layer channel to send on.</param>
         /// <returns></returns>
-        public virtual bool Send<T>(T msg, int channelId = Channels.DefaultReliable) where T: IMessageBase
+        public virtual bool Send<T>(T msg, int channelId = Channels.DefaultReliable) where T : IMessageBase
         {
             // pack message and send
             byte[] message = MessagePacker.Pack(msg);
+            NetworkDiagnostics.Send(msg, channelId, message.Length, 1);
```

Comment:
maybe "OnSend" is better? it looks like diagnostics are sending something to somewhere